.vscode/settings.json

```diff
@@ -42,5 +42,5 @@
         "unaugmented"
     ],
     "editor.detectIndentation": true,
-    "editor.insertSpaces": false, // Use detectIndentation instead
+    "editor.insertSpaces": true, // Use detectIndentation instead
```

Comment:
i was having problems with my editor defaulting to tabs, this seemed to help. editor.detectIndentation will still override it, but it must use insertSpaces when it is not sure, like in a new file